src/components/map/utils.ts

```diff
@@ -60,6 +60,10 @@ export const findStopPlaceAtClick = async (
 export const isFeaturePoint = (f: Feature): f is Feature<Point> =>
   f.geometry.type === 'Point';
 
+export const isClusterFeature = (
+  feature: Feature,
+): feature is Feature<Point, Cluster> => feature.properties?.cluster;
```

Comment:
Based on what is checked by this utility method we can't say with certainty that the `Feature` is of type `Point`, just that it has properties of type `Cluster`. In reality we might know that every `Feature` with `Cluster` properties also are of type `Point`, but I think it is that the utility methods are free of presumptions like this.

Maybe make the check `=> isFeaturePoint(feature) && feature.properties?.cluster`?